astroquery/simbad/core.py

```diff
@@ -335,42 +336,51 @@ class SimbadClass(BaseVOQuery, SimbadBaseQuery):
     def __init__(self):
         super().__init__()
         self._VOTABLE_FIELDS = self._VOTABLE_FIELDS.copy()
-        self._mirror = conf.server
+        self._server = conf.server
         self._tap = None
 
     @property
-    def simbad_mirrors(self):
-        """Set of the two Simbad mirrors domains."""
-        return {'simbad.cds.unistra.fr', 'simbad.harvard.edu'}
-
-    @property
-    def mirror(self):
+    def server(self):
         """The Simbad mirror to use."""
-        return self._mirror
+        return self._server
 
-    @mirror.setter
-    def mirror(self, server: str):
+    @server.setter
+    def server(self, server: str):
         f"""Allows to switch server between Simbad mirrors.
 
         Parameters
         ----------
         server : str
-            It should be one of {self.simbad_mirrors}.
+            It should be one of {conf.servers_list}.
```

Comment:
`servers_list` is not in conf, you can say the full path of `astroquery.simbad.servers_list` or just leave it as `servers_list`.